src/coreclr/inc/gcinfo.h

```diff
@@ -13,7 +13,11 @@
 /*****************************************************************************/
 
 #include "daccess.h"
-#include "windef.h"     // For BYTE
+#ifndef FEATURE_NATIVEAOT
+#include "windef.h"     // For UINT32
+#else
+#define UINT32 uint32_t
```

Comment:
It would be best to do find&replace of these with standard C types. It is what we have done for other components shared with naot.